site/gatsby-site/src/components/checklists/RiskSections.js

```diff
@@ -0,0 +1,62 @@
+import React, { useEffect, useState } from 'react';
+import { Form } from 'formik';
+import { Button, Textarea, Spinner } from 'flowbite-react';
+import { Trans, useTranslation } from 'react-i18next';
+import { useMutation } from '@apollo/client';
+import { LocalizedLink } from 'plugins/gatsby-theme-i18n';
+import { useExpanded, useFilters, usePagination, useSortBy, useTable } from 'react-table';
+
+import Table, { DefaultColumnFilter, DefaultColumnHeader } from 'components/ui/Table';
+import { DELETE_CHECKLIST } from '../../graphql/checklists';
```

Comment:
This import and others look unused. Please check here and other files being checked in?